rai/node/wallet.cpp

```diff
@@ -1311,34 +1311,37 @@ void rai::wallet::init_free_accounts (MDB_txn * transaction_a)
 	}
 }
 
-rai::public_key rai::wallet::change_seed (MDB_txn * transaction_a, rai::raw_key const & prv_a)
+rai::public_key rai::wallet::change_seed (MDB_txn * transaction_a, rai::raw_key const & prv_a, uint32_t count_a)
 {
 	store.seed_set (transaction_a, prv_a);
 	auto account = deterministic_insert (transaction_a);
-	uint32_t count (0);
-	for (uint32_t i (1), n (64); i < n; ++i)
+	uint32_t count (count_a);
```

Comment:
Probably count can be declared already in
rai::public_key rai::wallet::change_seed (MDB_txn * transaction_a, rai::raw_key const & prv_a, uint32_t count)
It's not constant or shared variable